.pre-commit-config.yaml

```diff
@@ -63,7 +63,7 @@ repos:
         name: check-code-generation
         entry: scripts/pre-commit-check-code-generation
         language: script
-        files: (pkg/graph/schema.graphql|pkg/graph/schema.resolvers.go|src/queries/*.ts)
+        files: (pkg/graph/schema.graphql|pkg/graph/schema.resolvers.go|src/queries/*.ts|pkg/cedar/intake/system_intake_schema.json)
```

Comment:
I need to include the Swagger yaml file in this, so the hook triggers on changes to that file.

Reply:
I wonder if it might be more obvious to a first-time onlooker what's happening if we just broke this out into another `repo` block that calls `scripts/gen_intake_schema` directly, rather than looping it in with `pre-commit-check-code-generation`?

I could probably go either way, but just was noodling on that!

Reply:
Good idea, done in https://github.com/CMSgov/easi-app/pull/1431/commits/2ea28e742a1895c7d834753b52c4cc709c098c1c.